gobblin-core/src/main/java/gobblin/converter/jdbc/JdbcEntryDatum.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.converter.jdbc;
+
+import com.google.common.base.Preconditions;
+
+
+public class JdbcEntryDatum {
+  private final String columnName;
+  private final Object val;
+  private int byteSize;
+
+  public JdbcEntryDatum(String columnName, Object val) {
+    this.columnName = Preconditions.checkNotNull(columnName);
+    this.val = Preconditions.checkNotNull(val);
+    this.byteSize = val.toString().getBytes().length;
```

Comment:
Is it necessary to control the size of the INSERT query based on the size? For now can't it just be based on the number on the number of values?

I agree its probably not a bottleneck now, but it may be a performance problem in the future. It may just be better to remove the buffer size restriction altogether unless there is a hard need for it. We can always add it back later when we have to time to think through how to implement it.